code-challenges.test.js

```diff
@@ -27,8 +35,51 @@ const colors2 = [
 ]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+//Test variable with different data types
+const colors3 = [6, "black", "white", true, null]
+//Expected output(can be in any order): ['white', true, 'black', null ]
+
+describe("shuffleArray", () => {
+  const expected1 = ["yellow", "blue", "pink", "green"]
```

Comment:
I am not too sure how this test is written will produce a pass without the original given variables